### PR TITLE
feat: Add Native tokens Tab to address pages

### LINKED ISSUE #1078: [Task]: Add native tokens tab to all address pages
### Task description

Add native tokens tab to all address pages

### PR DESCRIPTION
**Pull request by brancoder**
# Description of change

resolves #1078 

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: client/src/helpers/nova/hooks/useAccountAddressState.ts [hunk 1]
```diff
@@ -43,7 +48,8 @@ export const useAccountAddressState = (address: AccountAddress): [IAccountAddres
     );
 
     const { accountOutput, isLoading: isAccountDetailsLoading } = useAccountDetails(network, address.accountId);
-    const { totalBalance, availableBalance } = useAddressBalance(network, state.accountAddressDetails, accountOutput);
+    const { totalBalance, availableBalance } = useAddressBalance(network, state.addressDetails, accountOutput);
+    const [addressBasicOutputs, isBasicOutputsLoading] = useAddressBasicOutputs(network, state.addressDetails?.bech32 ?? null);
```
Comment (msarcev) [inline client/src/helpers/nova/hooks/useAccountAddressState.ts:52]: What about other outputs ? Afaik all `CommontOutput` (SDK-nova) can have the `NativeTokenFeature`.

File: client/src/app/components/nova/address/section/native-tokens/Asset.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
+/* eslint-disable jsdoc/require-param */
+/* eslint-disable jsdoc/require-returns */
+import { FoundryOutput, MetadataFeature, FeatureType, Irc30Metadata } from "@iota/sdk-wasm-nova/web";
+import { Validator as JsonSchemaValidator } from "jsonschema";
+import React, { ReactElement, useEffect, useState } from "react";
+import { AssetProps } from "./AssetProps";
+import tokenSchemeIRC30 from "~assets/schemas/token-schema-IRC30.json";
+import { useFoundryDetails } from "~helpers/stardust/hooks/useFoundryDetails";
```
Comment (msarcev) [inline client/src/app/components/nova/address/section/native-tokens/Asset.tsx:8]: Wrong import here (stardust)

File: client/src/app/components/nova/address/section/native-tokens/Asset.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                updateTokenInfo(metadata);
+            }
+        }
+    }, [isWhitelisted, foundryDetails]);
+
+    const updateTokenInfo = (metadata: MetadataFeature): void => {
+        const validator = new JsonSchemaValidator();
+
+        try {
+            const tokenInfo = JSON.parse(Converter.hexToUtf8(metadata.data)) as Irc30Metadata;
```
Comment (msarcev) [inline client/src/app/components/nova/address/section/native-tokens/Asset.tsx:40]: Let's use the `hexToUtf8` from SDK nova here (to move away from the iota.js stuff).

File: client/src/app/components/nova/address/section/native-tokens/Asset.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+/* eslint-disable jsdoc/require-returns */
+import { FoundryOutput, MetadataFeature, FeatureType, Irc30Metadata } from "@iota/sdk-wasm-nova/web";
+import { Validator as JsonSchemaValidator } from "jsonschema";
+import React, { ReactElement, useEffect, useState } from "react";
+import { AssetProps } from "./AssetProps";
+import tokenSchemeIRC30 from "~assets/schemas/token-schema-IRC30.json";
+import { useFoundryDetails } from "~helpers/stardust/hooks/useFoundryDetails";
+import { useTokenRegistryNativeTokenCheck } from "~helpers/stardust/hooks/useTokenRegistryNativeTokenCheck";
+import { Converter } from "~helpers/stardust/convertUtils";
+import Spinner from "../../../../Spinner";
```
Comment (msarcev) [inline client/src/app/components/nova/address/section/native-tokens/Asset.tsx:11]: This could be `import Spinner from "~/app/components/Spinner";`

Comment (msarcev) [reply client/src/app/components/nova/address/section/native-tokens/Asset.tsx:11]: There are others like this in the PR

**Review by msarcev — CHANGES_REQUESTED** [review]

File: client/src/helpers/nova/hooks/useAccountAddressState.ts (hunk 1 above)
Comment (brancoder) [reply client/src/helpers/nova/hooks/useAccountAddressState.ts:52]: On Nova only Basic and Foundry outputs can have NativeTokenFeature

**Comment by msarcev** [comment]
Seams like we lost the behaviour of "pre-selecting" the first tab somehow
image

**Pull request merged**